tremor-connectors/src/impls/http/utils.rs

```diff
@@ -13,6 +13,96 @@
 // limitations under the License.
 
 use either::Either;
+use futures::Stream;
+use hyper::body::Bytes;
+use std::pin::Pin;
+use std::task::Poll;
+
+/// A body of an HTTP request or response.
+pub struct Body {
+    pub(crate) data: Pin<Box<dyn Stream<Item = Result<Vec<u8>, std::io::Error>> + Send>>,
+}
+unsafe impl Send for Body {}
```

Comment:
This can be extremely dangerous, we should avoid it.

Reply:
Agree. Want to see CI passing tests before I attempt to refactor out Body. WSL isn't behaving well for me yet for unit tests. Integration tests pass.
As hyper now exposes `Request<Incoming>` and `Response<Incoming>` this shouldn't be too problematic now that the other issues like BodyExt are resolved.